Source/WebKit/UIProcess/API/glib/WebKitImagePrivate.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2025 Igalia S.L.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public License
+ * along with this library; see the file COPYING.LIB.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#pragma once
+
+#include "WebKitImage.h"
+
+guint webkit_image_hash(GIcon*);
+gboolean webkit_image_equal(GIcon*, GIcon*);
+
+GInputStream* webkit_image_load(GLoadableIcon*, int, char**, GCancellable*, GError**);
+void webkit_image_load_async(GLoadableIcon*, int, GCancellable*, GAsyncReadyCallback, gpointer);
+GInputStream* webkit_image_load_finish(GLoadableIcon*, GAsyncResult*, char**, GError**);
```

Comment:
This header does not seem to be needed. Virtual function implementations for GObjects are typically `static` to the file that implements a class, in this case `WebKitImage.cpp`.